pages/index.vue

```diff
@@ -2,7 +2,7 @@
   <main>
     <HomeHeroSection />
     <!-- <HomeIntroduction /> -->
```

Comment:
can we remove this commented out code?

Reply:
Resolved at https://github.com/arcana-network/website/pull/100/commits/8c7018051c9987cc08d2e705b386d93f5db59e3a